integrationtest/src/it/scala/io/gearpump/integrationtest/checklist/RestServiceSpec.scala

```diff
@@ -142,18 +143,64 @@ class RestServiceSpec extends TestSpecBase {
 
   "cluster information" should {
     "retrieve 1 master for a non-HA cluster" in {
+      // setup
+      val expectMastersCount = cluster.getMasterHosts.size
```

Comment:
Should use `expected` not `expect` for a variable. 

`expecto` is used for method, which means the method will expect particular behavior. If the behavior is unexpected, an exception will be thrown.